rules/falco_rules.yaml

```diff
@@ -3333,6 +3333,17 @@
   priority: WARNING
   tags: [host, container, mitre_execution, mitre_defense_evasion, T1036.005, T1059.004]
 
+- rule: Drop and execute new binary in container
+  desc:
+    Detect if an executable not belonging to the base image of a container is being executed.
+    The drop and execute pattern can be observed very often after an attacker gained an initial foothold.
+    is_exe_upper_layer filter field only applies for container runtimes that use overlayfs as union mount filesystem.
+  condition: >
+    spawned_process and container and proc.is_exe_upper_layer=true
+  output: Executing binary not part of base image (user=%user.name user_loginuid=%user.loginuid comm=%proc.cmdline exe=%proc.exe container_id=%container.id image=%container.image.repository, proc.name=%proc.name, proc.sname=%proc.sname, proc.pname=%proc.pname, proc.aname[2]=%proc.aname[2], exe_flags=%evt.arg.flags, proc.exe_ino=%proc.exe_ino, proc.exe_ino.ctime=%proc.exe_ino.ctime, proc.exe_ino.mtime=%proc.exe_ino.mtime proc.exe_ino.ctime_duration_proc_start=%proc.exe_ino.ctime_duration_proc_start, container.start_ts=%container.start_ts)
```

Comment:
- maybe also`%user.uid`
- highly recommending also adding `%proc.exepath` and `%proc.tty` so see if it was more from an interactive session and lastly `%evt.res`
- also putting `%proc.sid` and `%proc.vpgid` up for discussion also useful for sessionization of several rules

Reply:
Completely agree with everything on this. 
On the very last point, I didn't see any shadowing for the rule, but however given its importance is better to be safe and put it somewhere at the top of the file. I am doing this!